build.gradle

```diff
@@ -111,6 +111,10 @@ dependencies {
     compileOnly(javadocJDKFiles)
     testCompile(javadocJDKFiles)
 
+    // Library for configuration:
+    compile 'org.aeonbits.owner:owner:1.0.9'
+//    compile 'org.aeonbits.owner:owner:1.0.10-SNAPSHOT'
```

Comment:
I'm deleting the snapshot.  I just had it in there to check that it would with the latest and greatest (it does).